module_4/gift_certificates_system/controller/src/main/java/com/epam/esm/controller/util/PackageName.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.esm.controller.util;
+
+public final class PackageName {
```

Comment:
Please remove

Reply:
`PackageName.class` was renamed to `PackagePath.class` and used in the `EsmApplicationRunner.class`